README.md

```diff
@@ -12,11 +12,15 @@ This DSL relies on the Ruby [pact-mock_service][pact-mock-service] gem to provid
 * Google users group: https://groups.google.com/forum/#!forum/pact-support
 * Twitter: [@pact_up](https://twitter.com/pact_up)
 
-### Getting Started (with Karma, Jasmine and the pact-mock_service gem)
+### Installing pact-mock-service
 
-1. Install the [pact-mock_service](https://github.com/bethesque/pact-mock_service) ruby gem
+* If on Windows, please refer to the [Installing pact-mock-service on Windows](https://github.com/bethesque/pact-mock_service/wiki/Installing-the-pact-mock_service-gem-on-Windows) *
 
-   The easiest way is to add `gem 'pact-mock_service', '~> 0.2.3.pre.rc1'` to your `Gemfile` and run `bundle install`
+1. You must install [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) first.
```

Comment:
@bethesque I added the Gemfile back in, but I'm leaving the README how it is.  It might be good Ruby practice, but this isn't for people that write Ruby.  As of yet, I haven't had the need of more than one version of pact-mock-service.